tasks/main.yml

```diff
@@ -0,0 +1,46 @@
+---
+- name: Instantiate systemd-networkd network devices
+  template:
+    src: netdev.j2
+    dest: /etc/systemd/network/{{ netdev.key }}.netdev
+    owner: systemd-network
+    mode: 0640
+  loop: "{{ systemd_network_netdevs | dict2items }}"
+  loop_control:
+    loop_var: netdev
+  notify:
+    - Reload systemd configuration
+    - Restart systemd-networkd
+
+- name: Instantiate systemd-networkd networks
+  template:
+    src: network.j2
+    dest: /etc/systemd/network/{{ network.key }}.network
+    owner: systemd-network
```

Comment:
mode?

Reply:
The default mode on Debian 10 (0644) is fine. Should I explicitly state it?

Reply:
I thought maybe you forgot it, because you explicitly set it for the netdev files